eventing/channels/connecting-channels-sinks.adoc

```diff
@@ -16,4 +16,4 @@ include::modules/serverless-creating-subscription-admin-web-console.adoc[levelof
 
 [id="next-steps_connecting-channels-sinks"]
 == Next steps
-* Configure xref:../../eventing/serverless-event-delivery.adoc#serverless-configuring-event-delivery-examples_serverless-event-delivery[event delivery parameters] that are applied in cases where an event fails to be delivered to an event sink.
\ No newline at end of file
+* Configure xref:../../eventing/serverless-event-delivery.adoc#serverless-configuring-event-delivery-examples_serverless-event-delivery[event delivery parameters] that are applied in cases where an event fails to be delivered to an event sink.
```

Comment:
By right for accessibility reasons, the link metadata should state the title of the linked document.